meta-oe/recipes-support/cyaml/cyaml_1.0.bb

```diff
@@ -0,0 +1,27 @@
+SUMMARY = "Cyaml library"
+DESCRIPTION = "LibCYAML is a C library for reading and writing structured YAML documents. It is written in ISO C11 and licensed under the ISC licence."
```

Comment:
I am not sure if license info and C std are needed in description. Instead add more about functionality if you wish

Reply:
Okay done